packages/server/dataloader/azureDevOpsLoaders.ts

```diff
@@ -0,0 +1,495 @@
+          // if (!accessToken) return undefined
+          // const manager = new AzureDevOpsServerManager(accessToken)
+          // const auth = await parent.get('freshAzureDevOpsAuth').load({teamId, userId})
+          // console.log(`auth - ${auth}`)
+          // if (!auth) return []
+          // const {accessToken} = auth
+          // if (!accessToken) return undefined
+          const auth = await parent
+            .get('teamMemberIntegrationAuths')
+            .load({service: 'azureDevOps', teamId, userId})
+
+          if (!auth) {
+            return undefined
+          }
+
+          const provider = await parent.get('integrationProviders').loadNonNull(auth.providerId)
+
+          if (!provider) {
+            return undefined
+          }
+
+          const manager = new AzureDevOpsServerManager(auth, provider as IntegrationProviderAzureDevOps)
+
+          // const manager = new AzureDevOpsServerManager(accessToken)
+          const restResult = await manager.getAllUserWorkItems(null, false)
+          const {error, workItems} = restResult
+          if (error !== undefined || workItems === undefined) {
+            console.log(error)
+          } else {
+            returnWorkItems.push(...workItems)
+          }
+          return returnWorkItems
+        })
+      )
+      return results.map((result) => (result.status === 'fulfilled' ? result.value : undefined))
+    },
+    {
+      ...parent.dataLoaderOptions,
+      cacheKeyFn: (key) => `${key.teamId}:${key.userId}`
+    }
+  )
+}
+
+export const azureDevUserInfo = (
+  parent: RootDataLoader
+): DataLoader<TeamUserKey, AzureUserInfo | undefined, string> => {
+  return new DataLoader<TeamUserKey, AzureUserInfo | undefined, string>(
+    async (keys) => {
+      const results = await Promise.allSettled(
+        keys.map(async ({userId, teamId}) => {
+          // const auth = await parent.get('freshAzureDevOpsAuth').load({teamId, userId})
+          // if (!auth) return undefined
+          // const {accessToken} = auth
+          // if (!accessToken) return undefined
+          const auth = await parent
+            .get('teamMemberIntegrationAuths')
+            .load({service: 'azureDevOps', teamId, userId})
+
+          if (!auth) {
+            return undefined
+          }
+
+          const provider = await parent.get('integrationProviders').loadNonNull(auth.providerId)
+
+          if (!provider) {
+            return undefined
+          }
+
+          const manager = new AzureDevOpsServerManager(auth, provider as IntegrationProviderAzureDevOps)
+
+          // const manager = new AzureDevOpsServerManager(accessToken)
+          const restResult = await manager.getMe()
+          const {error, azureDevOpsUser} = restResult
+          if (error !== undefined || azureDevOpsUser === undefined) {
+            console.log(error)
+            return undefined
+          }
+          const user = azureDevOpsUser as unknown as AzureDevOpsUser
```

Comment:
-1 cast seems unnecessary, `azureDevOpsUser` is already of type `AzureDevOpsUser` here

Reply:
Modified to just expand on ...azureDevOpsUser in the returned object